test/common/utxobased/engine/engine.spec.ts

```diff
@@ -335,7 +340,7 @@ describe('engine.spec', function () {
               done() // Can be "done" since the promise resolves before the event fires but just be on the safe side
             }
           })
-          engine.startEngine().catch(e => {
+          engine.startEngine({ seenTxCheckpoint: '' }).catch(e => {
```

Comment:
Shouldn't this be undefined, instead of a blank string?

If the core types don't allow that, then we need to fix the core.